app/views/container_service/_main.html.haml

```diff
@@ -3,6 +3,8 @@
   .col-md-12.col-lg-6
     = render :partial => "shared/summary/textual", :locals => {:title => _("Properties"), :items => textual_group_properties}
     = render :partial => "shared/summary/textual_multilabel", :locals => {:title => _("Port Configurations"), :items => textual_group_port_configs}
+    = render :partial => "shared/summary/textual", :locals => {:title => _("Labels"), :items => textual_container_labels}
+    = render :partial => "shared/summary/textual", :locals => {:title => _("Selector"), :items => textual_container_selectors}
```

Comment:
If @simon3z was reacting the to same thing I was, then it's because the original screenshots show "Selectors" (plural).  But, I saw follow-up PR discussions where this was changed to "Selector".

Singular seems fine to me, as long as the intention is to really only have one.